samples/radio_test_bm/radio_test_subcommands.rst

```diff
@@ -165,7 +165,8 @@ Wi-Fi radio test subcommands
      - | <val> (Min: -10, Max: 10)
      - 0
      - Configuration
-     - Transmit tone frequency in the range of -10 MHz to 10 MHz.
+     - Tone frequency in the range of -10 MHz to 10 MHz with a resolution of 1 MHz.
+       The transmit tone frequency will be offset from the center frequency of the selected channel.
```

Comment:
Let's hold of on this for now till https://github.com/nrfconnect/nrf70-bm/pull/35#issuecomment-2581128134 is concluded, we can merge the other change in this PR.